src/style.css

```diff
@@ -1 +1,99 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --main-color: #00acdc;
+}
+
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+  width: 100%;
+}
+
+header {
+  width: 100%;
+  padding-right: 50px;

```

Comment:
```suggestion
  padding: 0 50px;

```

Reply:
than it would have also padding-left 50px, but there is no need in this